themes/openy_themes/openy_rose/templates/field_collection_item/field-collection-item--field-grid-columns--full.html.twig

```diff
@@ -0,0 +1,46 @@
+{#
+/**
+ * @file
+ * Default theme implementation for field collection items.
+ *
+ * Available variables:
+ * - content: An array of comment items. Use {{ content }} to print them all,
+ *   or print a subset such as {{ content.field_example }}. Use
+ *   {% hide(content.field_example) %} to temporarily suppress the printing
+ *   of a given element.
+ * - title: The (sanitized) field collection item label.
+ * - url: Direct url of the current entity if specified.
+ * - page: Flag for the full page state.
+ * - attributes: HTML attributes for the surrounding element.
+ *    Attributes include the 'class' information.
+ * - content_attributes: HTML attributes for the content element.
+ *
+ * @see template_preprocess()
+ * @see template_preprocess_entity()
+ *
+ * @ingroup themeable
+ */
+#}
+{%
+set classes = [
+  'grid-item',
+  'field-collection-item',
+  'field-collection-item--name-' ~ field_collection_item.name|clean_class,
+  'field-collection-item--view-mode-' ~ field_collection_item.view_mode|clean_class,
+]
+%}
+<div{{ attributes.addClass(classes) }}>
+  <div class="content"{{ content_attributes }}>
+    <div class="inner">
+      {% if content.field_prgf_column_class.0 %}
+        <i class="fa fa-{{ content.field_prgf_column_class.0 }}"></i>
+        <h2 {% if content.field_prgf_column_class.0 %}class="with-icon"{% endif %}>{{ content.field_prgf_clm_headline }}</h2>
+      {% else %}
+        {{ content.field_prgf_clm_icon }}
+        <h2 {% if content.field_prgf_clm_icon.0 %}class="with-icon"{% endif %}>{{ content.field_prgf_clm_headline }}</h2>
```

Comment:
Use set instruction with ternary operators to get values from the fields with default fallback
Using direct data in if prohibited by best practices (without defaults debugging in a future is a bit tough)
Example
```twig
{% set color = content.field_blog_color ? content.field_blog_color : 'red' %}
```